etcdmgmt/server.go

```diff
@@ -20,9 +20,7 @@ var etcdInstance = struct {
 
 // GetNewEtcdConfig will return reference to embed.Config object. This
 // is to be passed to embed.StartEtcd() function.
-func GetNewEtcdConfig() (*embed.Config, error) {
-
-	//TODO: Read local stored config to handle glusterd2 restarts
+func GetNewEtcdConfig(readConf bool) (*embed.Config, error) {
```

Comment:
Let `GetNewEtcdConfig` do just that, get a new etcd config. Let main handle reading or generating a new config.

Also move this to `utils.go`.